frontend/public/components/dashboard/details-card/detail-item.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import { OverlayTrigger, Tooltip } from 'patternfly-react';
+
+import { LoadingInline } from '../../utils';
+
+export const DetailItem: React.FC<DetailItemProps> = React.memo(({ title, value, isLoading }) => {
+  const description = value ? (
+    <OverlayTrigger
+      overlay={<Tooltip id={_.uniqueId('tooltip-for-')}>{value}</Tooltip>}
+      placement="top"
+      trigger={['hover', 'focus']}
+      rootClose={false}
+    >
+      <span>{value}</span>
+    </OverlayTrigger>
+  ) : (
+    '-'
+  );
+  return (
+    <React.Fragment>
+      <dt className="co-details-card__item-title">{title}</dt>
+      <dd className="co-details-card__item-value">{isLoading ? <LoadingInline /> : description}</dd>
```

Comment:
I haven't seen how this looks, but I suspect having a loading icon for every detail item is too much animation.

Again we should consider skeleton while the widget is loading.

Reply:
Yes, there is going to be quite a few loadings through the Overview Tab. We will be adding skeletons as a follow up.